### PR TITLE
data:Gawade15

### PR DESCRIPTION
**Pull request by Gawade15**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4753"></a>

### CODE REVIEW

**Review by krshkun — CHANGES_REQUESTED** [review]
The directory name should be lowercase, and the file extension should be json. Otherwise it's awesome

File: Data/Gawade15 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Komal Gawade",
+  "displayStatsPublic": true,
```
Comment (priyankarpal) [inline Data/Gawade15:3]: Currently you don't need to add this code 
```suggestion
```

File: Data/Gawade15 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "type": "personal",
+  "bio": " Learning Web developer ,  working on DSA , Exploring Technology",
+  "avatar":"https://github.com/Gawade15.png",
```
Comment (priyankarpal) [inline Data/Gawade15:6]: in a recent update, you don't need to add an avatar code.
```suggestion
  
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey please rename from `Data/Gawade15` to `Data/Gawade15.json`.

File: Data/Gawade15 (hunk 1 above)
Comment (swagataroy30) [inline Data/Gawade15:3]:
```suggestion
```

File: Data/Gawade15 (hunk 2 above)
Comment (swagataroy30) [inline Data/Gawade15:6]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar, it will auto-pick your GitHub profile picture and also we don't need to add `"displayStatsPublic": true`: anymore. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: Data/Gawade15 (hunk 1 above)
Comment (kumarsonsoff3) [inline Data/Gawade15:3]: As of our latest update, we no longer need the `displayStatsPublic` property. 

```suggestion
```

File: Data/Gawade15 (hunk 2 above)
Comment (kumarsonsoff3) [inline Data/Gawade15:6]: Also `avatar` key is no longer needed, it will auto-pick from your GitHub profile.

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
A few changes are required as **priyankarpal** suggested
Your file should be in data directory, to do so, you need to rename your [file here](https://github.com/Gawade15/LinkFree/edit/Gawade15-patch-1/Data/Gawade15) to `data/Gawade15.json`. 

- Change `Data/Gawade15`  - >  `data/Gawade15.json` here. 
- Remove the `avatar` & `displayStatsPublic` properties from your json, as these fields are no longer required.

Let us know if you'd any confusion. 

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Still it's not in the right directory. It should be exact `data/Gawade15.json`, `D` in data should be a small letter `d`. Follow the exact steps suggested above by me. 

Let us know if you need any help! :)

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**